tests/batch/views.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+from django.urls import reverse
+
+
+def test_batch_edit_translations_no_user(client):
+    """If there are no logged in users, the view redirects to the login page.
+    """
+    response = client.post(reverse('pontoon.batch.edit.translations'))
+    assert response.status_code == 302
+
+
+@pytest.mark.django_db
+def test_batch_edit_translations_bad_request(client, member0, locale0):
```

Comment:
@adngdb I would use `pytest.mark.parametrize` here because there's a lot of duplicated assertions - what do you think?

Reply:
I can't do that here sadly, it conflicts with using fixtures. I need to have the fixtures data in the parametrization decorator, which is not possible.